README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Join the chat at https://gitter.im/username-availability-checker/Lobby](https://badges.gitter.im/username-availability-checker/Lobby.svg)](https://gitter.im/username-availability-checker/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Check if a username can or cannot be taken on a bunch of popular social websites (such as Twitter, Instagram, Soundcloud, Github etc.) 
+Check if a username can or cannot be taken on a bunch of popular social websites (such as Twitter, Instagram, Soundcloud, Github etc.)
```

Comment:
Try to keep unrelated changes to a minimum. If you'd like, you can create a new commit with these spacing changes.

Reply:
Looks like atom automatically removes unneeded spaces at end of every lines. I'll revert those changes and add only necessary changes.